schemas/json/layout/layout.schema.v1.json

```
@@ -600,6 +610,41 @@
         }
       }
     },
+    "tableColumnOptions": {
+      "title": "Column options",
+      "description": "Column options for specified header.",
+      "type": "object",
+      "properties": {
+        "width":{
+          "title": "Width",
+          "description": "Width of cell in % or 'auto'. Defaults to 'auto'",
+          "type": "string",
+          "pattern": "^[0-9][0-9]%$|^100%$|^auto$"
```

We should probably also allow single-digit percentages
```suggestion
          "pattern": "^([0-9]{1,2}%|100%|auto)$"
```

---

Yes, of course! Nice catch 😁